package com.evergent.CoreJAVA.ExceptionalHandling;

public class ExceptionDemo2 {
//	String nameString="null";
	//public void myData() {
//		try {
//			System.out.println("one");
//			System.out.println(nameString.length());
//			System.out.println("End");
//		}
//		catch(NullPointerException e) {
//			System.out.println("I can handle "+e);
//		}
	//}
 public static void main(String[] args) {
//	ExceptionDemo2 ed2=new ExceptionDemo2();
//	ed2.myData();
	 String nameString="null";
	 try {
			System.out.println("one");
			System.out.println(nameString.length());
			System.out.println("End");
		}
		catch(NullPointerException e) {
			System.out.println("I can handle "+e);
		}
	 
}
}
